lib/google_drive/session.rb

```diff
@@ -418,18 +418,47 @@ def collection_by_url(url)
 
     alias folder_by_url collection_by_url
 
-    # Creates new spreadsheet and returns the new GoogleDrive::Spreadsheet.
+    # Creates a sub-folder with given title. Returns GoogleDrive::Collection
+    # object.
+    def create_subcollection(title, file_properties = {})
```

Comment:
Name it `create_collection` instead. "Subcollection" sounds weird since it's a method of a session. The comment should also say "Creates a folder at the top level".

Reply:
Ah, sorry, you are right!

Reply:
done